### PR TITLE
chore(ct): use sticky test server if available

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: packages/playwright-core/src/utils/network.ts [hunk 1]
```diff
@@ -169,6 +169,37 @@ export function createHttpsServer(...args: any[]): https.Server {
...
+      headers: { Accept: '*/*' },
+      rejectUnauthorized: !ignoreHTTPSErrors
+    }, res => {
+      res.resume();
+      const statusCode = res.statusCode ?? 0;
+      onLog?.(`HTTP Status: ${statusCode}`);
+      resolve(statusCode);
+    }, error => {
+      if ((error as NodeJS.ErrnoException).code === 'DEPTH_ZERO_SELF_SIGNED_CERT')
+        onStdErr?.(`[WebServer] Self-signed certificate detected. Try adding ignoreHTTPSErrors: true to config.webServer.`);
```
Comment (dgozman) [inline packages/playwright-core/src/utils/network.ts:196]: This error message seems overly specific for `utils/network.ts`

**Review by dgozman — APPROVED** [review]

File: packages/playwright-core/src/utils/network.ts (hunk 1 above)
Comment (pavelfeldman) [reply packages/playwright-core/src/utils/network.ts:196]: it is probably still applicable here

File: packages/playwright-ct-react17/package.json [hunk 2]
```diff
@@ -33,6 +33,7 @@
     "@vitejs/plugin-react": "^4.2.1"
   },
   "bin": {
-    "playwright": "cli.js"
+    "playwright": "cli.js",
+    "pw-react": "cli.js"
```
Comment (dgozman) [inline packages/playwright-ct-react17/package.json:37]:
```suggestion
    "pw-react17": "cli.js"
```

File: packages/playwright-ct-vue2/package.json [hunk 3]
```diff
@@ -36,6 +36,7 @@
     "vue": "^2.7.14"
   },
   "bin": {
-    "playwright": "cli.js"
+    "playwright": "cli.js",
+    "pw-vue": "cli.js"
```
Comment (dgozman) [inline packages/playwright-ct-vue2/package.json:40]:
```suggestion
    "pw-vue2": "cli.js"
```

**Review by dgozman — APPROVED** [review]

**Pull request merged**